charts/gardener/charts/application/templates/rbac-user.yaml

```diff
@@ -170,30 +224,30 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 rules:
-- apiGroups:
-  - ""
-  resources:
-  - configmaps
-  - serviceaccounts
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - garden.sapcloud.io
-  resources:
-  - shoots
-  - secretbindings
-  - quotas
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - core.gardener.cloud
-  resources:
-  - plants
-  verbs:
-  - get
-  - list
-  - watch
+  - apiGroups:
```

Comment:
Why has this been changed?

Reply:
Happened during Rebas, thanks for finding it!